src/unstract/sdk/file_storage/env_helper.py

```diff
@@ -32,7 +32,7 @@ def get_storage(storage_type: StorageType, env_name: str) -> FileStorage:
             and credentials configured in the env
         """
         try:
-            file_storage_creds = json.loads(os.environ.get(env_name))
+            file_storage_creds = json.loads(os.environ.get(env_name, ""))
```

Comment:
I think below code should need instaed of default empty string
```
file_storage_creds = json.loads(os.environ.get(env_name, "{}"))
```
But anyway, This default value might not help here. If it getting empty or None here the next line `37` will raise key error. right? So it would be better if we handle such exception here and raise meaning full error message or meaning full exception.

Reply:
Thank you. Changed.
Actually the exception is caught and an error message is displayed with the right format based in your feedback in the last PR. Pls look for it in lines 49-51 below.